app/scripts/models/auth_brokers/base.js

```diff
@@ -89,17 +94,44 @@ define(function (require, exports, module) {
     },
 
     /**
-     * initialize the broker with any necessary data.
+     * Initialize the broker with any necessary data.
+     *
      * @returns {Promise}
      */
     fetch () {
       return p().then(() => {
         this._isForceAuth = this._isForceAuthUrl();
         this.importSearchParamsUsingSchema(QUERY_PARAMETER_SCHEMA, AuthErrors);
+
+        if (this.hasCapability('fxaStatus')) {
+          return this._fetchFxaStatus();
+        }
       });
     },
 
     /**
+     * Request FXA_STATUS info from the UA.
+     *
+     * @returns {Promise} resolves when complete.
+     */
+    _fetchFxaStatus () {
+      const TEST_REQUEST_DELAY_MS = this.isAutomatedBrowser() ? 200 : 0;
+
+      const channel = this._notificationChannel;
+      return p().delay(TEST_REQUEST_DELAY_MS)
+        // OAuth reliers will have `service` set to the client_id, Sync to `sync`
```

Comment:
I was just thinking this comment doesn't really provide any value. I'll just rip it out instead of causing more confusion. The comment on lines 125-127 are the informative ones.